.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+version: "cygwin-{build}"
+
+environment:
+  CBUILD: x86_64-pc-linux-gnu
+  SCRIPT: generic/build
+  IMAGEROOT: /cygdrive/c/projects/%APPVEYOR_PROJECT_NAME%/image
+  matrix:
+    - ARCH: x86
+      CYGWIN: C:\Cygwin
+      CHOST: i686-w64-mingw32
+    - ARCH: x86_64
+      CYGWIN: C:\Cygwin64
+      CHOST: x86_64-w64-mingw32
+
+build_script:
+  - "%CYGWIN%\\setup-%ARCH%.exe -g -q -P mingw64-i686-openssl,mingw64-x86_64-openssl,cygwin-devel,unzip,dos2unix"
+  - "%CYGWIN%\\bin\\bash -lc 'cp `which true` /usr/bin/man2html'"
```

Comment:
I haven't found an elegant solution. 
I'd leave it for the community :) someone will come with good idea, I guess